### DISCUSSION TITLE
Need Bengali Language in Speech to Text

### DISCUSSION THREAD

**Discussion by Ntf-Sadnan** [deepgram/community#739] [Feature Requests]
Need Bengali Language in Speech to Text

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment]
@Ntf-Sadnan  I'll pass this onto our product team for consideration.

**Reply by sofwanmd13** [reply]
@jpvajda any updates so far? Really in need of Deepgram’s support in Bengali Language for Speeech to Text 🙏

**Reply by jpvajda** [reply]
Sorry @sofwanmd13  no updates on our end.
